src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -188,7 +188,18 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
 
   useEffect(() => {
     if (cameraAsset.hostname) {
-      getPresets({ onSuccess: (resp) => setPresets(resp.data) });
+      getPresets({
+        onSuccess: (resp) => setPresets(resp.data),
+        onError: (resp) => {
+          resp instanceof AxiosError &&
+            resp.response?.data?.errors &&
+            resp.response?.data?.errors.map((error: { message: string }) => {
```

Comment:
```suggestion
            resp.response?.data?.errors.forEach((error: { message: string }) => {
```
As we are not returning anything why not just use forEach instead of map

also change it in the other file too

Reply:
Resolved!